webapp/src/app/containers/home/home.component.ts

```diff
@@ -1,8 +1,53 @@
-import {Component} from '@angular/core';
+import {Component, Inject} from '@angular/core';
+import { AngularFireAuth } from '@angular/fire/compat/auth';
+import { FormBuilder, FormGroup } from '@angular/forms';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.scss']
 })
-export class HomeComponent {}
+export class HomeComponent {
+  
+  userForm!: FormGroup;
```

Comment:
If you actually put the form setup (lines 22-25) into the constructor, you don't need this `!` non-null assertion operator.